### PR TITLE
Disable Pylint

### PR DESCRIPTION
**Pull request by jart**
Pylint is a nice tool, but hasn't been that helpful for TensorBoard.

- See https://travis-ci.org/tensorflow/tensorboard/jobs/376968526 where it
  takes more than two minutes to run, representing >35% of Travis CI latency.

- We typically have to wait until the very end of a Travis run before we get
  Pylint results, because we run it in a really hackish way. This is also
  likely the cause of its poor performance and has resulted in surprising
  breakages, such as the one we encountered in #1206.

- Pylint has a very different point of view on style / best practices compared
  with Google and IntelliJ.

- TensorBoard doesn't work with the latest version of Pylint.

IntelliJ is good enough, in my personal opinion. I'd be open to keeping Pylint
if it gave quick feedback as part of the normal Bazel build process. I looked
into the difficulty of doing that and it was discouraging.

### CODE REVIEW

**Comment by jart** [comment]
Note: I also used this as an opportunity to tidy up our Travis config a bit.

File: .travis.yml [hunk 1]
```diff
@@ -41,51 +44,48 @@ before_install:
...
 
-  - |
-    bazel \
-      --output_base="${HOME}/.bazel-output-base" \
-      --batch \
-      --host_jvm_args=-Xmx500m \
-      --host_jvm_args=-Xms500m \
-      test \
-      tensorboard/... \
-      --verbose_failures \
```
Comment (nfelt) [inline .travis.yml]: Do we not want `--verbose_failures` and `--test_verbose_timeout_warnings` any more?

Comment (jart) [reply .travis.yml]: Yes indeed we do. I'm also seeing if I can shave some latency off the Bazel install.

**Comment by jart** [comment]
Wow @nfelt, that last Travis run took 3.5 minutes! https://travis-ci.org/tensorflow/tensorboard/builds/382976663

Also we might want to consider adopting [flake8](http://flake8.pycqa.org/en/latest/) in the future. It has less zeal than Pylint, goes faster, and Wenzel uses it for PyBind11.

**Pull request merged**